File: core/src/test/java/cn/enaium/mineconf/java/MineConfLoaderTest.java
package cn.enaium.mineconf.java;

import cn.enaium.mineconf.MineConfLoader;
import org.junit.jupiter.api.Test;

/**
 * @author Enaium
 */
public class MineConfLoaderTest {
    @Test
    public void loader() {
        System.setProperty("user.dir", "run");
        MineConfLoader.load();
    }
}

File: core/src/test/kotlin/cn/enaium/mineconf/kt/MineConfLoaderTest.kt
package cn.enaium.mineconf.kt

import cn.enaium.mineconf.core.MineConfLoader
import org.junit.jupiter.api.Test

/**
 * @author Enaium
 */
class MineConfLoaderTest {
    @Test
    fun loader() {
        System.setProperty("user.dir", "run")
        MineConfLoader.load()
    }
}
